cpp/src/lance/format/schema.h

```diff
@@ -75,6 +75,9 @@ class Schema final {
   /// Top level fields;
   const std::vector<std::shared_ptr<Field>> fields() const { return fields_; }
 
+  /// Count the number of all fields, including nested fields.
```

Comment:
so this is the number of leaves in the schema tree right?

Reply:
This is for the size of the entire tree, including parent nodes.